cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -445,7 +445,7 @@ class alignas(16) column_device_view : public detail::column_device_view_base {
   __device__ T element(size_type element_index) const noexcept
   {
     size_type index       = element_index + offset();  // account for this view's _offset
-    char const* d_strings = d_children[strings_column_view::chars_column_index].data<char>();
+    char const* d_strings = static_cast<char const*>(_data);
```

Comment:
Should we change the next line (`offsets`) to be consistent with this?

Reply:
The `offsets` still need to be retrieved from the `d_children`. I'm not sure what you are suggesting.